### PR TITLE
Push Notification Concept RFC

### PR DESCRIPTION
**Pull request by TelegramSam**
Signed-off-by: Sam Curren <telegramsam@gmail.com>

### CODE REVIEW

**Review by blu3beri — APPROVED** [review]
It looks good to me, just one small note that might help maintainers of a public mediator.

File: concepts/0757-push-notification/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+8. Notification Recipient (mobile app) then connects to the Notification Sender (mediator) and receives waiting messages.
+
+In spite of the flow similarities between the push notification platforms, the implementations, libraries used, and general code paths vary substantially. 
+Each push notification method is described in it's own protocol. This allows the protocol to fit the specific needs and terminology of the notification method it enables. Feature Discovery can be used between the Notification Sender and the Notification Recipient to discover push notification compatibility.
+
+### Public Mediators
+
+Some push notification methods require matching keys or secrets to be used in both sending and receiving notifications. This requirement makes these push notification methods unusable by public mediators.
+
+Public mediators SHOULD only implement push notification methods that do not require sharing secrets or keys with application implementations.
```
[the remark this answers is not in the document]
Comment (TelegramSam) [reply concepts/0757-push-notification/README.md:42]: I'm not sure about that just yet, so was planning to leave it for an update.

[the remark this answers is not in the document]
Comment (blu3beri) [reply concepts/0757-push-notification/README.md:42]: Yeah makes sense.

**Review by swcurran — APPROVED** [review]
Approved by the Aries Working Group 2022.11.09.

**Pull request merged**